### PR TITLE
Top navigation

### PR DESCRIPTION
**Pull request by funionnn**
Move sidebar navigation into a top bar

@vernalkick 

---
Update: On Stacks#show: 
screen shot 2015-05-21 at 7 34 01 pm
(Specific links can be added/changed, this is more to just fix the styles)

### CODE REVIEW

File: app/views/stacks/_header.html.erb [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+  <p><%= stack.repo_owner %></p>
+  <h1><a href="<%= stack_path(@stack) %>"><%= stack.repo_name %> <small><%= stack.environment %></small></a></h1>
+  <% if stack.deploy_url.present? %>
+    <%= link_to stack.deploy_url, stack.deploy_url, :target => '_blank', :class => 'deploy-url' %>
+  <% end %>
+<% end %>
+
+
+<% content_for :primary_navigation do %>
+  <%= link_to 'Refresh', stack_refresh_path(@stack), method: 'post', data: {tooltip: 'Refresh statuses and commits'}, class: "btn secondary" %>
```
Comment (funionnn) [inline app/views/stacks/_header.html.erb]: This tooltip needs to go below the button: 

screen shot 2015-05-21 at 5 45 56 pm

**Comment by funionnn** [comment]
@vernalkick @gmalette @byroot 
We should merge this!

**Comment by vernalkick** [comment]
I tweaked the design a bit, so I'm making a few final changes and then we can press that button!

**Comment by funionnn** [comment]
Perfect, go for it when you're ready :+1:

**Comment by byroot** [comment]
:+1: merge whenever you want!

**Pull request merged**